tests/unit/calendar/calendar_core.js

```diff
@@ -139,7 +139,7 @@ asyncTest( "keyboard handling", function() {
 			.focusGrid( element )
 			.simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
 		setTimeout(function() {
-			$( ":focus" ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
+			$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
```

Comment:
I guess that something for anther PR as we have a quite some more glitches to fix in the unit tests.